### PR TITLE
[KYUUBI #2761] Flaky Test: engine.jdbc.doris.StatementSuite - test select

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _Why are the changes needed?_

close https://github.com/apache/incubator-kyuubi/issues/2761#issue-1250326421
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhaomin1423** [comment]
I have update the docker image. It may be fix the problem. @ulysses-you

File: externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ services:
       - "9030"
 
   doris-be:
-    image: zhaomin1423/doris-be:1.0.0
+    image: zhaomin1423/doris-be:1.0.0-0.1
```
Comment (turboFei) [inline externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml:26]: could you provide the Dockerfile?

Comment (zhaomin1423) [reply externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml:26]: add it to project?

Comment (turboFei) [reply externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml:26]: yes，it is helpful.

Comment (ulysses-you) [reply externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml:26]: yeah, it looks like a black box for Kyuubi if no more detials. We should add some comments to explain at least.

Comment (zhaomin1423) [reply externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml:28]: added the image link included Dockerfiles to comment.

**Review by turboFei — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**